app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SessionControlController.kt

```diff
@@ -372,6 +372,10 @@ class DefaultSessionControlController(
             TopSite.Type.PINNED -> metrics.track(Event.TopSiteOpenPinned)
         }
 
+        if (url == SupportUtils.GOOGLE_XX_URL) {
+            metrics.track(Event.TopSiteOpenGoogleSearchAttribution)
+        }
```

Comment:
Are there tests for the `DefaultSessionControlController` that we can add for this?

Reply:
There should be